package.json

```diff
@@ -43,10 +43,12 @@
     }
   },
   "dependencies": {
-    "date-fns": "^2.3.0",
+    "@popperjs/core": "^2.10.1",
+    "date-fns": "^2.23.0",
     "memoize-one": "^5.1.0",
     "react-datepicker": "^4.2.1",
     "react-onclickoutside": "^6.7.1",
+    "react-popper": "^2.2.5",
```

Comment:
Ok so we gain 4 KB (nice), but we loose

- `react-popper` (5.4 KB)
- `@popperjs/core` (20 KB)

A total of 25 KB - so 21 KB more.

See https://bundlephobia.com/package/@popperjs/core@2.10.1 and https://bundlephobia.com/package/react-popper@2.2.5.

Am I missing something?

Reply:
Alright but then I'd recommend to use `^2.9.2` for the `@popperjs/core` version range, to have the same base quantifier (resolved anyway to 2.10.1 currently).

Reply:
Good catch, done!